index.html

```diff
@@ -0,0 +1,114 @@
+                <p class="dates">JUN, 2017–AUG, 2017</p>
+                <p>Created, structured, and taught: 8 week curriculum for expansive ideation and concept development for
+                    rising freshmen at various art institutes in the US.
+                </p>
+            </div>
+            <div class="content03">
+                <h2>Skills</h2>
+                <h3>Graphic Design</h3>
+                <ul class="u-list-reset">
+                    <li>Concept Ideation</li>
+                    <li>Typeface Design</li>
+                    <li>Social Media Branding</li>
+                    <li>Editorial & Publication</li>
+                    <li>Digital Photography</li>
+                    <li>Narrartive Wireframing</li>
+                </ul>
+                <h3>Sound Design</h3>
+                <ul class="u-list-reset">
+                    <li>Studio Recording</li>
+                    <li>Mixing & Mastering</li>
+                    <li>Stage Setting</li>
+                    <li>Live Performance</li>
+                    <li>Sound Installation</li>
+                    <li>Foley & Sound Effects</li>
+                </ul>
+            </div>
+            <div class="content04">
+                <h2>Language</h2>
+                <h3>Bilingual: Enlgish/Korean</h3>
+                <h2>Education</h2>
+                <h3>Maryland Institute College of Art(MICA)</h3>
+                <p class="dates">AUG, 2016–Current</p>
+                <p>BFA candidate in Graphic Design<br>Expected May 2022
+                </p>
+                <p class="dates">JAN, 2019–AUG, 2020</p>
+                <p>LOA for the Republic of Korea Army<br>Enlisted KATUSA: Human Resource Specialist<br>2nd Infantry Division, Camp Humprheys
+                    
+                </p>
+            </div>
+
+            <div class="content05">
+                <h2>Softwares</h2>
+                <ul class="u-list-reset">
+                    <li>Indesign</li>
+                    <li>Illustrator</li>
+                    <li>After Effects</li>
+                    <li>Premiere Pro</li>
+                    <li>Audition</li>
+                    <li>Glyphs</li>
+                    <li>Ableton Live</li>
+                </ul>
+                <h2>Awards</h2>
+                <h3>MICA Scholarships</h3>
+                <p class="dates">AUG, 2016</p>
+                <ul class="u-list-reset">
+                    <li>Creative Vision Award</li>
+                    <li>Trustee Award</li>
+                    <li>Academic Excellence Scholarship</li>
+                    <li>Foundation Scholarship</li>
+                </ul>
+                <p class="dates">APR, 2017</p>
+                <ul class="u-list-reset">
+                    <li>Competitive Scholarship Award</li>
+                </ul>
+
+            </div>
+            <div class="content06">
+                <h2>Contact</h2>
+                <p>ijung01@mica.edu<br>+82)1028735284</p>
```

Comment:
Same note about replacing `br` with a list. Also, for an HTML resume, it also makes sense for your email address to be a link; you do that by using [`mailto`](https://css-tricks.com/snippets/html/mailto-links/):
```suggestion
                <p><a href="mailto:ijung01@mica.edu">ijung01@mica.edu</a><br>+82)1028735284</p>
```